cos-fleetshard-operator-debezium/src/test/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandControllerTest.java

```diff
@@ -208,7 +214,7 @@ void reify(ObjectNode connectorConfig, Consumer<KafkaConnect> kafkaConnectChecks
             .first()
             .isInstanceOfSatisfying(KafkaConnect.class, kc -> {
                 assertThat(kc.getSpec().getTemplate().getPod().getImagePullSecrets())
-                    .contains(DebeziumConstants.IMAGE_PULL_SECRET);
+                    .contains(CONFIGURATION.imagePullSecretsName());
```

Comment:
This should not be null, but the default value (`addon-pullsecret`), shouldn't it?